### DISCUSSION TITLE
Packages maintenance

### DISCUSSION THREAD

**Discussion by eviltik** [felixmariotto/three-mesh-ui#177] [Q&A]
Hi,

Just for testing, i've upgraded all npm packages, including threejs.

```
$ ncu --packageFile package.json -u
Upgrading C:\...\three-mesh-ui\package.json

 @types/three         ^0.136.1  =>  ^0.138.0
 eslint                ^7.25.0  =>   ^8.11.0
 html-webpack-plugin    ^4.5.2  =>    ^5.5.0
 webpack               ^4.46.0  =>   ^5.70.0
 webpack-cli           ^3.3.12  =>    ^4.9.2
 webpack-dev-server    ^3.11.2  =>    ^4.7.4

Run npm install to install new versions.
```

Then, i fixed args in `package.json`
`--env NODE_ENV=prod` (without the dot)

Then, `contentBase` attribute to `static` attribute in `webpack.prodConfig.js`.

I've tested all npm commands, no errors. All examples seem's fine. But perhaps i missed something ?

**Comment by swingingtom** [comment]
Hey @eviltik, thanks !

I started an upgrade last thursday I didn't finished yet, because I wanted to be sure to support more features.

Could I ask to review it and give your input when it's done ?

**Comment by eviltik** [comment]
No problem !

Btw https://github.com/eviltik/three-mesh-ui/commit/d4bde3d1aa0f85564004067436cc9e5a8c86aaa1

**Reply by swingingtom** [reply]
Hey @eviltik, sorry for the huge delay. If still okay for you to review, this is the PR #180.
Thank you in advance

**Comment by eviltik** [comment]
Separate examples for each type of builds is really cool for users.
I didn't notice something wrong and i retrieve globaly what i have on my side. I'm not an expert, but everything seem's fine in your PR.

**Reply by swingingtom** [reply]
Thanks for that lightning fast feedback !

**Reply by swingingtom** [reply]
> Separate examples for each type of builds is really cool for users.

Those are more for the development side, in order to be sure everything is fine. For users I was thinking about codesandboxes in order for them to show the proper paths 

```javascript
<script type="importmap">
  {
    "imports": {
      "three": "https://unpkg.com/three@0.132.2/build/three.module.js",
      "three-mesh-ui" : "https://unpkg.com/three-mesh-ui@6.4.0/build/three-mesh-ui.module.min.js"
      // instead of 
      // "three-mesh-ui" : "./three-mesh-ui.module.min.js"
    }
  }
</script>
```
